.github/next_release.md

```diff
@@ -20,6 +20,7 @@ Below is a list of changes:
 
 - Installation
      - CMake &ge; 3.15 is now required (was &ge; 3.8).
+     - Python &ge; 3.8 is now required (was &ge; 3.5), because of `importlib.metadata`.
```

Comment:
Maybe the use of type hints also requires 3.8? I don't remember.

Reply:
In Python 3.5 for [type hints](https://docs.python.org/3/library/typing.html)

Reply:
Ok. Note that they added new type hinting possibilities with almost every release, so it is hard to be sure we aren't using modern things that were absent in older releases we are not testing anymore. Anyway, let's ignore this.